config/core/configmaps/gc.yaml

```diff
@@ -48,9 +48,17 @@ data:
     stale-revision-timeout: "15h"
 
     # Minimum number of generations of revisions to keep before considering
-    # them for GC
+    # them for GC.
     stale-revision-minimum-generations: "20"
 
+    # Maximum number of generations of revisions to keep before considering
+    # them for GC.  If creation or staleness time-bounds are set, the system
+    # will ignore those times to achieve this cap. -1 to disable a maximum bound.
+    #
+    # ALPHA NOTE: Note this behavior will be applied only when the
+    #             'responsive-revision-gc' feature is enabled.
+    stale-revision-maximum-generations: "1000"
```

Comment:
Should we disable by default for BC?

Reply:
That probably makes lots of sense... since the feature is disabled by default, this would be invalid / ignored.